src/Tizen.Messaging/Tizen.Messaging.Messages/Message.cs

```diff
@@ -131,6 +131,21 @@ private void AddAddress(MessagesAddress address)
             }
         }
 
+        public int GetMessageCount(int mbox, int messageType)
+        {
+            int count;
+            Log.Error(Globals.LogTag, "GetMessageCount is Called");
+            int ret = Interop.Messages.GetMessageCount(_messageHandle, mbox, messageType, out count);
+            if (ret != (int)MessagesError.None)
+            {
+                Log.Error(Globals.LogTag, "Failed to get Message count, Error - " + (MessagesError)ret);
+                MessagesErrorFactory.ThrowMessagesException(ret, _messageHandle);
+                return 1;
```

Comment:
In C# we prefer an exception handling over error codes, which means that functions should focus on their purpose rather than returning success/failure value. The purpose of this method is to "get count", which you are currently NOT returning anyway (because count is assigned locally). Please rewrite this method to return count value, but don't use 'out' parameter as in native implementation.